Client/mods/deathmatch/logic/CClientModel.cpp

```diff
@@ -70,8 +70,50 @@ bool CClientModel::Allocate(ushort usParentID)
     return false;
 }
 
+bool CClientModel::MakeAtomicModel()
+{
+    if (!m_bWasConvertedToClump)
+        return false;
+
+    CModelInfo* pModelInfo = g_pGame->GetModelInfo(m_iModelID, true);
+
+    if (!pModelInfo->IsValid() || m_eModelType != eClientModelType::OBJECT)
```

Comment:
**GetModelInfo** can return **nullptr** what should be taken into account in this conditional expression.